scripting/include/srccoop/manager.inc

```diff
@@ -158,8 +221,9 @@ methodmap CoopManager
 		g_pSpawnOptions.Reset();
 
 		ItemInstancingManager.OnPlayerSpawned(pPlayer);
+
+		data.m_iSpawnedPlayerCount = GetRealClientCount(true);
```

Comment:
Technically this is already being counted in g_iPlayerCount.
If you want alive only, the second parameter has to be true.

Reply:
For this, I need a player count that updates only on player spawn. With `g_iPlayerCount`, it gets updated both on player connects and disconnects. Keeping this as is.